Makefile

```diff
@@ -0,0 +1,4 @@
+test:
+	@./tests/test_testimony.sh | sed -r "s/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[mGK]//g" | diff tests/sample_output.txt -
```

Comment:
Do you know that there is an option to disable testimony colored output? You can use it to make this logic simple.
Example:
`$ testimony auto tests --nocolor`
or
`$ testimony auto tests -n`